### DISCUSSION TITLE
discord integration

### DISCUSSION THREAD

**Discussion by MacOS-9** [86Box/86Box#1572] [Q&A] [answered]
So, i was going to enable discord integration in 86box, but there was no option? idk if its a bug or i did something wrong, but it is not there? I am using 86box-Debug, discord is opened and the discord game sdk dll is there

**Comment by dhrdlicka** [comment, marked as answer]
You need to download the latest build from http://ci.86box.net/ in order to use rich presence. Version 2.07 (presuming you are using that, since you didn't mention the version) does not include the functionality.

**Reply by MacOS-9** [reply]
i think i am using the latest version let me check

**Reply by MacOS-9** [reply]
it says i am using version 2.7.0.2008 in the 86box manager

**Reply by MacOS-9** [reply]
i tried downloading it from the website you linked but my browser wont download it for life it just gives up after a while

**Reply by MacOS-9** [reply]
ok i downloaded it using a download manager it worked thanks
